terraform/datadog/aws.tf

```diff
@@ -58,3 +58,34 @@ resource "datadog_monitor" "rds-failure" {
 
   tags = ["deployment:${var.env}", "service:${var.env}_monitors", "job:all"]
 }
+
+resource "datadog_monitor" "ec2-cpu-utilisation-warning" {
+  name                = "${format("%s CPU load has been moderately high for a long time on {{bosh-job.name}}/{{bosh-index.name}}", var.env)}"
+  type                = "metric alert"
+  query               = "${format("avg(last_2h):avg:aws.ec2.cpuutilization{deploy_env:%s} by {bosh-job,bosh-index} > 100", var.env)}"
```

Comment:
Am I reading this correctly? This will never fire as a critical alert, even though it's defined because the threshold is `> 100`? We're relying on the second monitor to fire critical alerts?

Reply:
Yeah, as mentioned in other comment. This monitor never fires critical. .... the "> 100" in the query looks odd, but it reads the warning level separately from the query :)